### PR TITLE
get more info

### PR DESCRIPTION
**Pull request by 4c0d3r**

### CODE REVIEW

File: namer/watchdog.py [hunk 1]
```diff
@@ -262,7 +262,10 @@ def create_watcher(namer_watchdog_config: NamerConfig) -> MovieWatcher:
     """
     logger.remove()
     level = 'DEBUG' if namer_watchdog_config.debug else 'INFO'
-    logger.add(sys.stdout, format="<green>{time:YYYY-MM-DD HH:mm:ss}</green> | <level>{level.icon} {level: <8}</level> | {message}", level=level, diagnose=namer_watchdog_config.diagnose_errors)
+    if platform.system().lower() != 'windows':
+        logger.add(sys.stdout, format="<green>{time:YYYY-MM-DD HH:mm:ss}</green> | <level>{level.icon} {level: <8}</level> | {message}", level=level, diagnose=namer_watchdog_config.diagnose_errors)
+    else:
+        logger.add(sys.stdout, format="<green>{time:YYYY-MM-DD HH:mm:ss}</green> | <level>{level: <8}</level> | {message}", level=level, diagnose=namer_watchdog_config.diagnose_errors)
```
Comment (DirtyRacer1337) [inline namer/watchdog.py:268]: What's point of that?

Comment (DirtyRacer1337) [reply namer/watchdog.py:268]: I'm using only windows and I don't have problem with icons

Comment (4c0d3r) [reply namer/watchdog.py:268]: cp1252 characters didn't print in logging in the last build.   Character's not supported.   Trying to debug the freezing tests (doesn't happen locally - so will be throwing crap at GA)

Comment (4c0d3r) [reply namer/watchdog.py:268]:
```
--- Logging error in Loguru Handler #7 ---
[807](https://github.com/ThePornDatabase/namer/actions/runs/3802665501/jobs/6468363430#step:10:808)
Record was: {'elapsed': datetime.timedelta(seconds=40, microseconds=489226), 'exception': None, 'extra': {}, 'file': (name='watchdog.py', path='D:\\a\\namer\\namer\\namer\\watchdog.py'), 'function': 'stop', 'level': (name='INFO', no=20, icon='\u2139\ufe0f'), 'line': 242, 'message': 'Exiting watchdog', 'module': 'watchdog', 'name': 'namer.watchdog', 'process': (id=5568, name='MainProcess'), 'thread': (id=852, name='MainThread'), 'time': datetime(2022, 12, 29, 19, 31, 46, 53770, tzinfo=datetime.timezone(datetime.timedelta(0), 'Coordinated Universal Time'))}
[808](https://github.com/ThePornDatabase/namer/actions/runs/3802665501/jobs/6468363430#step:10:809)
Traceback (most recent call last):
[809](https://github.com/ThePornDatabase/namer/actions/runs/3802665501/jobs/6468363430#step:10:810)
  File "C:\Users\runneradmin\AppData\Local\pypoetry\Cache\virtualenvs\namer-dnQ_sQ65-py3.9\lib\site-packages\loguru\_handler.py", line 177, in emit
[810](https://github.com/ThePornDatabase/namer/actions/runs/3802665501/jobs/6468363430#step:10:811)
    self._sink.write(str_record)
[811](https://github.com/ThePornDatabase/namer/actions/runs/3802665501/jobs/6468363430#step:10:812)
  File "C:\Users\runneradmin\AppData\Local\pypoetry\Cache\virtualenvs\namer-dnQ_sQ65-py3.9\lib\site-packages\loguru\_simple_sinks.py", line 16, in write
[812](https://github.com/ThePornDatabase/namer/actions/runs/3802665501/jobs/6468363430#step:10:813)
    self._stream.write(message)
[813](https://github.com/ThePornDatabase/namer/actions/runs/3802665501/jobs/6468363430#step:10:814)
  File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\encodings\cp1252.py", line 19, in encode
[814](https://github.com/ThePornDatabase/namer/actions/runs/3802665501/jobs/6468363430#step:10:815)
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
[815](https://github.com/ThePornDatabase/namer/actions/runs/3802665501/jobs/6468363430#step:10:816)
UnicodeEncodeError: 'charmap' codec can't encode characters in position 35-36: character maps to <undefined>
```

Comment (4c0d3r) [reply namer/watchdog.py:268]: Also looks like logging may subltly effect timing as nothing failed on that push, rebuild now.

Comment (DirtyRacer1337) [reply namer/watchdog.py:268]: изображение
Last jobs, how this help if mac os and windows have same execution time? Or I don't get what you mean

Comment (4c0d3r) [reply namer/watchdog.py:268]: The logging may be preventing the deadlocking, or making it more rare in some of the tests.

Comment (DirtyRacer1337) [reply namer/watchdog.py:276]: Ok this is helped, https://github.com/ThePornDatabase/namer/pull/131/commits/d2ed87e96cfec0fb46073399ed380367e20cc19c

Comment (DirtyRacer1337) [reply namer/watchdog.py:276]: Played more, only one place where I get this is PyChram and problem was in their pytest analyser, they do `value.encode(self.encoding)` instead like `value.encode(self.encoding, errors='ignore')`

Comment (DirtyRacer1337) [reply namer/watchdog.py:276]: Fixed #132

**Pull request merged**